renderer/src/Animation.cpp

```diff
@@ -0,0 +1,71 @@
+#include "Animation.hpp"
```

Comment:
This file has a few interesting quirks:
- Java style curly braces
- Two-space indentation
- Missing single space before only some curly braces before function bodies